lib/matplotlib/axes/_base.py

```diff
@@ -1060,6 +1060,10 @@ def cla(self):
             self.yaxis.set_visible(yaxis_visible)
             self.patch.set_visible(patch_visible)
         self.stale = True
+        if(self.xaxis_inverted()):
```

Comment:
Is there a way to do this without the tests?

These should go above setting `self.stale = True`